redhat-developer/developers.redhat.com
# [DEVELOPER-5194] Disqus

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5194

### Verification Process
- Visit the DevOps page once this has been pushed (comments will not work locally, to avoid running up our Disqus API use)
- Look at or create the curated content, dynamic content, and dynamic content list assemblies
- Verify that on the dynamic content and curated content assemblies, a comment counter is visible and links to the comments section on the linked node
- Verify that recent comments appear in a sidebar next to the dynamic content list, as seen in the mockups
- Click the reply links on the latest comments bar. They should link to the particular comment on the relevant node.
- Visit an article, video resource, cheat sheet, and book. Scroll to just above the footer. There should be a comments section on each of these pages.

### Other Notes
I'm not sure if the Jira issue was changed or if I didn't notice when I started it, but the comment counters weren't supposed to be a part of this issue. They were supposed to be later work, but I ended up doing them as part of this task.
